.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Run Checks
+      run: ./gradlew check --stacktrace
```

Comment:
you can also add `--no-daemon`, do we also need to add `buid`?

Reply:
`check` depends on `build` so it's not required explicitly